programs/perpetuals/src/state/custody.rs

```diff
@@ -290,6 +287,9 @@ impl Custody {
     }
 
     pub fn unlock_funds(&mut self, amount: u64) -> Result<()> {
+        if self.is_virtual {
+            return Ok(());
```

Comment:
Should ideally throw to avoid conflict with later design since virtual asset custodies can never lock/unlock assets

Reply:
updated